deployment/ansible/roles/install_config_nginx/tasks/main.yml

```diff
@@ -63,12 +16,20 @@
 - name: Create Nginx configuration file
   ansible.builtin.template:
     src: nginx.conf.j2
-    dest: /etc/nginx/conf.d/default.conf
+    dest: /etc/nginx/nginx.conf
     owner: root
     group: root
     mode: '0644'
   register: nginx_conf
-  notify: Restart_nginx
+  tags: ["nginx"]
+
+- name: Create Nginx site configuration file
+  ansible.builtin.template:
+    src: default.conf.j2
+    dest: /etc/nginx/conf.d/default.conf
```

Comment:
```suggestion
/etc/nginx/sites-available/xxxx
```

Reply:
and don't forget to create symlink to /etc/nginx/sites-enabled/xxxx

Reply:
hope that properly done